docs/content/examples/tutorials/adguard.md

```diff
@@ -1,9 +1,91 @@
+# AdGuard Home with WG Easy and Watchtower (Docker Compose / Host Mode)
+
+## 📌 Description
+This setup runs **AdGuard Home**, **WG Easy** (WireGuard management UI) and **Watchtower** for automatic updates — all in **Docker host network mode**.
+
 ---
-title: AdGuard Home
----
 
-It seems like the Docs on how to setup AdGuard Home are not available yet.
+## 📂 Prepare Local Folders
+
+Before starting, create the required directories:
+
+```bash
+sudo mkdir -p /home/admin/docker/wireguard
+sudo mkdir -p /home/admin/docker/adguard/work
+sudo mkdir -p /home/admin/docker/adguard/conf
+mkdir -p /home/admin/docker/watchtower
```

Comment:
i would follow the same folder structure as the basic installation. where wg-easy is at `/etc/docker/containers/wg-easy`